CDP4ShellDialogs/ViewModels/ModelOpeningDialogViewModel.cs

```diff
@@ -144,6 +246,49 @@ private async Task ExecuteSelect()
             this.DialogResult = new BaseDialogResult(true);
         }
 
+        /// <summary>
+        /// Executes the SelectActiveIteration command
+        /// </summary>
+        /// <returns>
+        /// The <see cref="Task"/>.
+        /// </returns>
+        private async Task ExecuteSelectActiveIteration()
+        {
+            var stopWatch = Stopwatch.StartNew();
+            var activeIterationSetupRow = this.SelectedEngineeringModel.IterationSetupRowViewModels.FirstOrDefault(x => x.Thing.FrozenOn == null);
+
+            this.IsBusy = true;
+            this.LoadingMessage = "Loading the Active Iteration...";
+
+            if (activeIterationSetupRow != null)
+            {
+                if (!(activeIterationSetupRow.Thing.Container is EngineeringModelSetup modelSetup))
+                {
+                    throw new NullReferenceException($"The EngineeringModelSetup that iteration {activeIterationSetupRow.Thing.Iid} belongs to cannot be found.");
+                }
+
+                // Retrieve the Iteration from the IDal
+                var session = activeIterationSetupRow.Session;
+                var model = new EngineeringModel(modelSetup.EngineeringModelIid, session.Assembler.Cache, session.Credentials.Uri)
+                { EngineeringModelSetup = modelSetup };
+
+                var iteration = new Iteration(activeIterationSetupRow.Thing.IterationIid, session.Assembler.Cache, session.Credentials.Uri);
+
+                model.Iteration.Add(iteration);
+                var t = Task.Run(() =>
+                {
+                    session.Read(iteration, activeIterationSetupRow.SelectedDomain);
```

Comment:
Please clarify the second part related to "since the text on the dialog is related to Default Domain.." It is not clear to me. Do you refer to the selected domain of the active iteration? Which test on the dialog we referring here?

Reply:
> 
> 
> Please clarify the second part related to "since the text on the dialog is related to Default Domain.." It is not clear to me. Do you refer to the selected domain of the active iteration? Which test on the dialog we referring here?

The `User` class has a property called `defaultDomain` which can be 0..1. When a `Person` has a `defaultDomain` AND this `DomainOfExpertise` is  for that Person part of a `Participant`, then this is the domain which should used for the "open iteration action"  as a result of the double-click on the Engineering Model setup. 

My proposal, In case the Person does not have a default domain set, then the double click should result in selecting the first DomainOfExpertise based on the sort order (shortname) of the assigned domains-of-expertise.